../../pobj/netatalk-4.0.2/netatalk-4.0.2/meson.build:897:11: ERROR: Unknown variable "sparql".
../../pobj/netatalk-3.2.2/netatalk-3.2.2/etc/uams/meson.build:229:30: ERROR: Unknown variable "pam".

Index: meson.build
--- meson.build.orig
+++ meson.build
@@ -901,6 +901,7 @@ else
     endif
 endif
 
+if have_spotlight
 have_spotlight = (
     sparql.found()
     and indexer_found
@@ -908,6 +909,8 @@ have_spotlight = (
     and flex.found()
     and bison.found()
 )
+endif
+
 if have_spotlight
     cdata.set('WITH_SPOTLIGHT', 1)
 endif
@@ -1610,6 +1613,7 @@ uams_options = ''
 
 if not enable_pam
     have_pam = false
+    pam = []
 else
     if host_os != 'sunos'
         pam_paths = [
